pur/__init__.py

```diff
@@ -68,6 +68,8 @@
               help='Prevents updating nested requirements files.')
 @click.option('-s', '--skip', type=click.STRING, help='Comma separated list ' +
               'of packages to skip updating.')
+@click.option('--index-url', type=click.STRING, help='Comma separated list ' +
```

Comment:
Let's allow `--index-url` to be passed multiple times with each argument added to a list. Then we can have urls that include commas.